server/ai_live_video.go

```diff
@@ -87,8 +88,9 @@ func startTrickleSubscribe(ctx context.Context, url *url.URL, params aiRequestPa
 				clog.Infof(ctx, "Error reading trickle subscription: %s", err)
 				return
 			}
-			defer segment.Body.Close()
-			if _, err = io.Copy(w, segment.Body); err != nil {
+			clog.V(8).Infof(ctx, "trickle subscribe read data")
+
+			if err = copySegment(segment, w); err != nil {
```

Comment:
Just a heads up this probably conflicts with https://github.com/livepeer/go-livepeer/pull/3303 which tried to fix the same issue, but I like this solution so please keep it in